src/cpp/session/modules/automation/SessionAutomationRemote.R

```diff
@@ -26,55 +26,48 @@
 .rs.defineVar("automation.remotePrivateEnv", new.env(parent = .rs.toolsEnv()))
 .rs.defineVar("automation.remote", new.env(parent = emptyenv()))
 
-.rs.addFunction("automation.createRemote", function(mode = NULL)
+.rs.setVar("automation.remoteInstance", NULL)
```

Comment:
I want to try and simplify some of this; I feel like this is getting a bit on the unwieldy side. The intention though is to make it easy to get + set a global "singleton" remote instance, so that we can easily create and destroy RStudio automation agents (so that we can run subsets of tests in a "clean" RStudio session)